Missing sphinx-rtd-theme in Chapter 3 while running `make html`

Running  `poetry run make html` gave me this error:

```
Running Sphinx v4.0.1
WARNING: sphinx_rtd_theme (< 0.3.0) found. It will not be available since Sphinx-6.0

Theme error:
no theme named 'sphinx_rtd_theme' found (missing theme.conf?)
make: *** [html] Error 2
```

I fixed it by adding the missing dependency:

```
poetry add --dev sphinx-rtd-theme 
```

--- a/py-pkgs/03-how-to-package-a-python.ipynb
+++ b/py-pkgs/03-how-to-package-a-python.ipynb
@@ -558,7 +558,7 @@
     "First, we need to install `sphinx`, `nbsphinx`, and `ipykernel` as development dependencies using `poetry`. \n",
     "\n",
     "```{prompt} bash\n",
-    "poetry add --dev sphinx nbsphinx ipykernel sphinx-autoapi\n",
+    "poetry add --dev sphinx nbsphinx ipykernel sphinx-autoapi sphinx-rtd-theme\n",
     "```\n",
     "\n",
     "```{note}\n",

